pkg/odo/cli/preference/view.go

```diff
@@ -63,9 +63,19 @@ func (o *ViewOptions) Run(ctx context.Context) (err error) {
 func (o *ViewOptions) RunForJsonOutput(ctx context.Context) (result interface{}, err error) {
 	preferenceList := o.clientset.PreferenceClient.NewPreferenceList()
 	registryList := o.clientset.PreferenceClient.RegistryList()
+
+	var regList, newRegistryList []preference.Registry
+	if registryList != nil {
+		regList = *registryList
+	}
+	// Loop backwards here to ensure the registry display order is correct (display latest newly added registry firstly)
+	for i := len(regList) - 1; i >= 0; i-- {
+		newRegistryList = append(newRegistryList, regList[i])
+	}
```

Comment:
The CLI level should not have to handle this, it is the responsability of the business level to return the correct information. This way, we should not have to duplicate this code here and in `HumanReadableOutput`.

Also, there is a nice feature in go to exchange two values: `a, b = b, a`. You could use it to reverse the order of the array in place (an elegant solution here: https://stackoverflow.com/a/53736310).